recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ outputs:
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
         - {{ compiler('c') }}
+        - {{ stdlib("c") }}
       host:
         - python
         - pip
```

Comment:
```suggestion
        - pip
        - setuptools
```

Reply:
For what I've understood, setuptools need to be specified for Python 3.13 builds.
https://github.com/conda-forge/msgspec-feedstock/pull/51#issuecomment-2564169121

Reply:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1124445&view=logs&j=67d2a488-2689-5a3d-98d8-781873f29742&t=6eebd5c8-f4d8-5ebe-1b42-85bfa58284b2&l=1111
`ModuleNotFoundError: No module named 'setuptools'
`